en/flight_modes/takeoff.md

```diff
@@ -31,47 +31,91 @@ Parameter | Description
+- Launch/throw the vehicle (firmly) directly into the wind. You can also shake the vehicle first, wait till the motor spins up and throw only then.
+
+The Launch detector is affected by the following parameters:
+
+Parameter | Description
+--- | ---
+<span id="LAUN_ALL_ON"></span>[LAUN_ALL_ON](../advanced_config/parameter_reference.md#LAUN_ALL_ON) | Enable automatic launch detection. If disabled motors spin up on arming already.
+<span id="FW_LAUN_AC_THLD"></span>[FW_LAUN_AC_THLD](../advanced_config/parameter_reference.md#FW_LAUN_AC_THLD) | Acceleration threshold (acceleration in body-forward direction must be above this value).
+<span id="FW_LAUN_AC_T"></span>[FW_LAUN_AC_T](../advanced_config/parameter_reference.md#FW_LAUN_AC_T) | Trigger time (acceleration must be above threshold for this amount of seconds).
+<span id="FW_LAUN_MOT_DEL"></span>[FW_LAUN_MOT_DEL](../advanced_config/parameter_reference.md#FW_LAUN_MOT_DEL) | Delay from launch detection to motor spin up.
 
-:::note
-In addition to the behaviour discussed above there is also a launch detector that may block the launch sequence from starting until some condition is met.
-For catapult launch this is some acceleration threshold.
-:::
 
 <span id="runway_launch"></span>
 ### Runway Takeoff
 
-The *runway takeoff mode* has the following phases:
+Runway takeoffs can be used by vehicles with landing gear and and steerable wheel (only).
+You will first need to enable the wheel controller using the parameter [FW_W_EN](#FW_W_EN).
 
-1. **Throttle ramp**: Clamped to the runway (pitch fixed, no roll, and heading hold) until reach the minimum airspeed for takeoff ([FW_AIRSPD_MIN](#FW_AIRSPD_MIN) x [RWTO_AIRSPD_SCL](#RWTO_AIRSPD_SCL)).
-1. **Takeoff**: Increase pitch and continue until vehicle altitude > navigation altitude ([RWTO_NAV_ALT](#RWTO_NAV_ALT)).
-1. **Climbout**: Climb until altitude above ground level > [FW_CLMBOUT_DIFF](#FW_CLMBOUT_DIFF).
-   In this phase roll and heading restrictions are removed.
 
+Vehicle should be centered and aligned with runway when takeoff is initiated (either by arming in takeoff mode or by starting a mission with a takeoff mission item as active waypoint).
+The operator can "nudge" the vehicle while on the runway to help keeping it centered and aligned (see [RWTO_NUDGE](../advanced_config/parameter_reference.md#RWTO_NUDGE)).
 
-### Fixed Wing Takeoff Parameters
 
-Takeoff is affected by the following parameters:
+The *runway takeoff mode* has the following phases:
+1. **Throttle ramp**: Throttle is ramped up within [RWTO_RAMP_TIME](../advanced_config/parameter_reference.md#RWTO_RAMP_TIME) to [RWTO_MAX_THR](../advanced_config/parameter_reference.md#RWTO_MAX_THR).
+1. **Clamped to runway**: Pitch fixed, no roll and course hold until the rotation airspeed ([RWTO_ROT_AIRSPD](../advanced_config/parameter_reference.md#RWTO_ROT_AIRSPD)) is reached. The operator is able to nudge the vehicle left/right via yaw stick.

```

Comment:
not sure about calling it "course hold" - as it is actually following a line. maybe "controls to takeoff path"